gradle.properties

```diff
@@ -12,3 +12,5 @@ POM_DEVELOPER_ID=airbnb
 POM_DEVELOPER_NAME=Airbnb
 POM_DEVELOPER_EMAIL=android@airbnb.com
 POM_INCEPTION_YEAR=2017
+
+org.gradle.configureondemand=false
```

Comment:
Why set this explicitly? Isn't `false` the default at the moment?

Reply:
Unfortunately no, `3.1.3` AGP version does not support it.

I was getting this error:
`Configuration on demand is not supported by the current version of the Android Gradle plugin since you are using Gradle version 4.6 or above. Suggestion: disable configuration on demand by setting org.gradle.configureondemand=false in your gradle.properties file or use a Gradle version less than 4.6.`

Reply:
Updated!